force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -122,13 +166,12 @@ public with sharing class FlowControlLoopsCollections {
      * @return The sum of all integers up to the limit.
      */
     public static Integer sumUpToLimit(Integer intLimit) {
-        /* 
-         for (){
-            
+        Integer sumTotal = null;
+        for (Integer i = 0 ; i > intLimit ; i++){
+            sumTotal = i + sumTotal;
+            System.debug(sumTotal);
```

Comment:
The best practice is always to remove debugs from a commit. Don't let them get to prod.